src/Kunstmaan/NodeSearchBundle/Helper/HtmlSearchCrawler.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Kunstmaan\NodeSearchBundle\Helper;
+
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Class HtmlSearchCrawler.
```

Comment:
Docblock can be removed as it provides no "usefull" info

Reply:
It's a php cs fixer induced habit 🙄